pkg/controller/backup_controller.go

```diff
@@ -269,9 +273,16 @@ func (controller *backupController) processBackup(key string) error {
 
 	logContext.Debug("Running backup")
 	// execution & upload of backup
+	backupName := kubeutil.NamespaceAndName(backup)
+	backupScheduleName := kubeutil.LabelValue(backup, "ark-schedule")
+	controller.metrics.RegisterBackupAttempt(backupName, backupScheduleName)
```

Comment:
If there's no schedule, won't this generate a new metric per backup, e.g.
```
ark_server_backup_attempts_total{backupName="foo1", schedule=""} 1
ark_server_backup_attempts_total{backupName="foo2", schedule=""} 1
ark_server_backup_attempts_total{backupName="foo3, schedule=""} 1
```

Reply:
Yes, it will. I thought that is the behaviour @rosskukulinski was preferring, no?

Reply:
I don't think we want that

Reply:
I take back my initial answer. I am not sure if it will create a metric per backup. trying to get the correct answer

Reply:
OK.. after consulting with a collegue who knows more prom than me! we might just drop the backup name altogether and just have the schedule name.